integration-tests/src/test_loop/tests/bandwidth_scheduler.rs

```diff
@@ -1,6 +1,19 @@
-use std::collections::{BTreeMap, BTreeSet};
+//! Test bandwidth scheduler on a "real" chain.
+//! Start a single node with some shards and run a workload which sends a lot of cross-shard
+//! receipts. Measure bandwidth utilization and fairness and make sure that they're reasonable. It's
+//! a bit tricky to achieve high utilization. You need to have a lot of concurrent transactions, and
+//! even then most of the time some outgoing buffers will be empty, it's probably slowed down by
```

Comment:
I actually want the opposite. I want the congestion level to be as low as possible so that the shard accepts a lot of new transactions, which will put a lot of receipts in the outgoing buffers. The more receipts in the outgoing buffers, the better the throughput. I also want the scheduler to have a choice on where to send receipts, it shouldn't be limited to only the sender shard.